appveyor.yml

```diff
@@ -4,9 +4,15 @@ install:
 build: off
 matrix:
   fast_finish: true
+  exclude:
+    - platform: Ruby25-x64
+      LOCALE: ja-JP
+    - platform: Ruby24-x64
+      LOCALE: ja-JP
```

Comment:
Why are we skipping these?  Should we skip ja-JP testing altogether?  I don't see this documented in the PR comments or the ticket.

Reply:
I mentioned it in the commit message https://github.com/puppetlabs/puppet/pull/7494/commits/2d291ea57303f6f2a7d5ace6858297ce87d1d9fc and the PR summary above.

I'd prefer to not skip testing on Japanese altogether, as it provides some assurance we're not introducing regressions in internationalization. For example, that `fast-gettext` can still attempt to resolve localized strings and fallback to english. That said, testing on Japanese across 4 different ruby versions seems like overkill so I wanted to sparsen the matrix.

That said,  let me take this part out, as it looks like we should drop one of the axes in 6.4.x also.

Reply:
Done